packages/jest-matchers/src/__tests__/matchers-test.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails oncall+jsinfra
+ */
+
+'use strict';
+
+jest.disableAutomock();
+
+const expect = require('../index.js').expect;
```

Comment:
Also, can we call expect something different in this test, like `jestExpect`? It's confusing that below we are calling `expect` just normally.